egs/ljspeech/TTS/local/prepare_token_file.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# Copyright         2023  Xiaomi Corp.        (authors: Zengwei Yao)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This file reads the texts in given manifest and generate the file that maps tokens to IDs.
+"""
+
+import argparse
+import logging
+from collections import Counter
+from pathlib import Path
+from typing import Dict
+
+import g2p_en
+import tacotron_cleaner.cleaners
```

Comment:
Please add code about how to install `tacotron_cleaner`?

Reply:
Thanks a lot!